libtmux/server.py

```diff
@@ -319,8 +319,6 @@ def attached_sessions(self):
         """
         Return active :class:`Session` objects.
 
-        This will not work where multiple tmux sessions are attached.
```

Comment:
Hm, back in history, I'm wondering what I will thinking back then.

https://github.com/tmux-python/tmuxp/commit/1519deeed2d5299e34eb48d4b530e48bcde939db

I guess there really wasn't a good rationale back then.